roles/ceph-rgw/tasks/rgw_create_pools.yml

```diff
@@ -0,0 +1,74 @@
+---
+  - name: remove ec profile
+    command: "{{ container_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd erasure-code-profile rm {{ item.value.ec_profile }}"
+    with_dict: "{{ rgw_create_pools }}"
+    delegate_to: "{{ groups[mon_group_name][0] }}"
+    changed_when: false
+    run_once: true
+    register: result
+    when: 
+      - item.value.pool_type is defined  
+      - item.value.pool_type == 'ec'
+    ignore_errors: yes
```

Comment:
why?

Reply:
The reason we had this was if the playbook gets run again. When the playbook gets run again, it will try to rm the ec_profile and we didn't want it to fail if the ec_profile was already created and being used.